src/features/amUI/userRightsPage/DelegationModal/ResourceInfo.tsx

```diff
@@ -140,9 +144,21 @@ export const ResourceInfo = ({ resource, toParty, onDelegate }: ResourceInfoProp
         ),
       };
 
-      delegateRights(delegationInput).then(() => {
-        onDelegate();
-      });
+      delegateRights(delegationInput)
+        .then(() => {
+          openSnackbar({
+            message: t('delegation_modal.success_message', { name: toParty.name }),
+            variant: SnackbarMessageVariant.Success,
```

Comment:
```suggestion
            variant: SnackbarMessageVariant.Success,
            duration: SnackbarDuration.long,
```
I personally think the normal duration is a bit too short. Consider changing it to long.

Reply:
Agreed 👍